client/lib/service-worker/service-worker.js

```diff
@@ -35,25 +35,35 @@ self.addEventListener( 'push', function( event ) {
 		self.registration.showNotification( notification.msg, {
 			tag: 'note_' + notification.note_id,
 			icon: notification.icon,
-			timestamp: notification.note_timestamp
+			timestamp: notification.note_timestamp,
+			data: notification
+		} ).then( function () {
+			if ( notification.note_opened_pixel ) {
+				fetch( notification.note_opened_pixel, { mode: 'no-cors' } ).catch( function( err ) {
+					console.log( 'Could not load the pixel %s', notification.note_opened_pixel );
```

Comment:
In Chrome, the service worker has its own console, which can be accessed via `chrome://serviceworker-internals/`.
In Firefox, the browser console (`Shift` + `Command` + `J`)